src/api/admin.ts

```diff
@@ -129,6 +129,20 @@ export interface VenueInput_v2
   end_utc_seconds?: number;
 }
 
+export interface WorldProps {
+  id: string;
```

Comment:
I don't think the id should be here by default because it's not mirrored in the data. I think we should follow the `WithId` pattern here

Reply:
Fixed in: https://github.com/sparkletown/sparkle/pull/2295/commits/563960b617e371ddfc9d465b3428bdae64fafb0d